packages/perseus/src/util/graphie.test.ts

```diff
@@ -1320,6 +1321,40 @@ describe("Graphie drawing tools", () => {
             expect(onSetDrawingAreaAvailable).toHaveBeenCalledWith(false);
         });
     });
+
+    describe("getMousePx", () => {
+        it.each([
+            [{left: 0, top: 0}, {pageX: 10, pageY: 10}, [10, 10]],
+            [{top: 10, left: 20}, {pageX: 40, pageY: 40}, [20, 30]],
+        ])(
+            "should return the mouse position in pixel coordinates relative to graph",
+            (offset, mouseEvent, expectedMousePx) => {
```

Comment:
`offset` is the screen coordinates of the top-left corner of the graph, right? Could its name reflect that? Something like `graphPosition` might work.